packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
 <Project ToolsVersion="16.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\runtime.win10-arm.Microsoft.Net.UWPCoreRuntimeSdk.2.2.10\build\runtime.win10-arm.Microsoft.Net.UWPCoreRuntimeSdk.props" Condition="Exists('..\packages\runtime.win10-arm.Microsoft.Net.UWPCoreRuntimeSdk.2.2.10\build\runtime.win10-arm.Microsoft.Net.UWPCoreRuntimeSdk.props')" />
```

Comment:
>Import [](start = 3, length = 6)

Is it correct to put these props files here?
Normally in C++ project they must go below.

Reply:
This is where VS put it...